src/vic3_world/countries/vic3_country.h

```diff
@@ -56,6 +65,8 @@ class Country
    [[nodiscard]] const commonItems::Color& GetColor() const { return color_; }
    [[nodiscard]] const std::optional<int>& GetCapitalState() const { return capital_state_; }
    [[nodiscard]] bool IsDecentralized() const { return country_type_ == "decentralized"; }
+   /// Is the country a recognized or colonial nation
+   [[nodiscard]] bool IsRecognized() const { return !IsDecentralized() && country_type_ != "unrecognized"; }
```

Comment:
Add tests for the two different way this can be false (ItemsCanBeInput counts as one of them) and also it being true.